### PR TITLE
November 2019 test-fix ready for review and QC

### PR DESCRIPTION
**Pull request by kylebunting**
The November 2019 test-fix is ready for review and QC.

### CODE REVIEW

File: Hands-on lab/HOL step-by-step - Data Platform upgrade and migration.md [hunk 1]
```diff
@@ -886,15 +882,15 @@ In this exercise, you will install Oracle XE on your Lab VM, load a sample datab
 
 ### Task 4: Install dbForge Fusion tool
 
-In this task, you will install a third-party extension to Visual Studio to enable interaction with, and script execution for, Oracle databases in Visual Studio Community 2017 Edition.
+In this task, you will install a third-party extension to Visual Studio to enable interaction with, and script execution for, Oracle databases in Visual Studio 2017 Community 2017 Edition.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Data Platform upgrade and migration.md]: Hey there!  This line doesn't make sense.  Should it be Visual Studio 2019 Community Edition?

File: Hands-on lab/HOL step-by-step - Data Platform upgrade and migration.md [hunk 2]
```diff
@@ -954,29 +950,22 @@ WWI has provided you with a copy of their application, including a database scri
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Data Platform upgrade and migration.md]: Please see line 943 (task 5, number 10) Instructions are select Visual Studio 2017 and select OK.  Image is of VS2017 too...but we've got VS2019 throughout rest of WS, I think this one got missed.  Thanks

File: Hands-on lab/Before the HOL - Data Platform upgrade and migration.md [hunk 3]
```diff
@@ -59,35 +58,45 @@ In the Before the hands-on lab exercise, you will set up your environment for us
...
 
-1. In the [Azure portal](https://portal.azure.com/), select **+Create a resource**, enter "visual studio" into the Search the Marketplace box and select Visual Studio.
+1. In the [Azure portal](https://portal.azure.com/), select the **Show portal menu** icon and then select **+Create a resource** from the menu.
 
-    ![+Create a resource is selected in the Azure navigation pane, and "visual studio community" is entered into the Search the Marketplace box. Visual Studio Community 2017 (latest release) on Windows Server 2016 (x64) is selected in the results.](./media/create-resource-visual-studio-on-windows-server-2016.png "Create Windows Server 2016 with Visual Studio Community 2017")
+    ![The Show portal menu icon is highlighted and the portal menu is displayed. Create a resource is highlighted in the portal menu.](media/create-a-resource.png "Create a resource")
 
-2. On the Visual Studio blade, select the Select a software plan drop down list and then select **Visual Studio Community 2017 (latest release) on Windows Server 2016 (x64)** from the list.
+2. Enter "visual studio 2019" into the Search the Marketplace box and select Visual Studio.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL - Data Platform upgrade and migration.md]: This was originally Enter "visual studio" into the Search.....
The alt-text and the image both showed "visual studio 2019" so I added the 2019 to the instructions. Change back if that was incorrect.

File: Hands-on lab/HOL step-by-step - Data Platform upgrade and migration.md (hunk 1 above)
Comment (kylebunting) [reply Hands-on lab/HOL step-by-step - Data Platform upgrade and migration.md]: Yep, I clearly messed that one up! I've fixed it. Commit is below.

File: Hands-on lab/HOL step-by-step - Data Platform upgrade and migration.md (hunk 2 above)
Comment (kylebunting) [reply Hands-on lab/HOL step-by-step - Data Platform upgrade and migration.md]: I fixed this.

File: Hands-on lab/Before the HOL - Data Platform upgrade and migration.md (hunk 3 above)
Comment (kylebunting) [reply Hands-on lab/Before the HOL - Data Platform upgrade and migration.md]: Should be "visual studio 2019." Your change is correct. Included in my commit below.

**Pull request merged**